src/som/primitives/ExceptionsPrims.java

```diff
@@ -103,4 +103,5 @@ public final Object doException(final SBlock body, final SBlock ensureHandler) {
       }
     }
   }
+
```

Comment:
unnecessary  change

Reply:
Don't know how to remove that. CheckStyle added it for us I think.